db/migrate/20160315014311_create_posts.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePosts < ActiveRecord::Migration
+  def change
+    create_table :posts do |t|
+      t.string :post_title
+      t.string :post_datestamp
+      t.string :post_text
+      t.string :tag_title
+      t.string :tag_text
```

Comment:
This is an advanced topic, but `tags` are a classic example of a many-to-many relationship (and thus are usually not indicated as attributes in this way). Let me know if you'd like to talk about this.